packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -476,10 +494,11 @@ class Scrollable extends Component {
 					touchableProps,
 					verticalScrollbarProps
 				}) => (
-					<ScrollableSpotlightContainer
+					<div
```

Comment:
@MikyungKim and I discussed about the comment above. We're going not to fix the comment because <div> element has customized attributes and scrollbars. So it is better to remain the code.